wire-runtime-swift/src/main/swift/ProtoCodable/ProtoCodable.swift

```diff
@@ -1,24 +1,18 @@
 /*
- * Copyright 2020 Square Inc.
+ * Copyright (C) 2020 Square, Inc.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- * http://www.apache.org/licenses/LICENSE-2.0
+ *      https://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-// MARK: -
-
-/**
- A protocol which defines a type as being deserializable from protocol buffer data.
- */
```

Comment:
hmmmm

Reply:
This would definitely be a regression.

Reply:
Can you `import Foundation` before defining the protocol?

Reply:
The thing is looking for some text to make the difference between some copyright text and some real code. It looks like Swift allows whatever to be there. Kotlin would have package/import/public as the delimiter.
I'm not sure what to set for swift, maybe I could add `//` but doesn't feel right.

Reply:
So my suggestion, which is a hack, would be to `import Foundation` after the copyright.
While you don't _really_ need Foundation here, there's zero harm in importing it.

Reply:
got it, I did that